cylc/flow/config.py

```diff
@@ -751,22 +751,26 @@ def _check_implicit_tasks(self) -> None:
         queue config, unless allowed by config."""
         if self.implicit_tasks:
             print_limit = 10
-            implicit_tasks_str = '\n    * '.join(
+            impl_tasks_str = '\n    * '.join(
```

Comment:
I can't tell whether `impl` expands to `implicit` or `implementation` or something else. I'd prefer the original term.